modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLSubscriptionSender.java

```diff
@@ -60,7 +60,7 @@ protected Boolean hasSubscribePermission(
 			ServiceProxyFactory.newServiceTrackedInstance(
 				ModelResourcePermission.class, DLSubscriptionSender.class,
 				"_dlFolderModelResourcePermission",
-				"(model.class.name=" + DLFolder.class.getName() + ")", true);
+				"(model.class.name=" + DLFolder.class.getName() + ")", false);
```

Comment:
@W2jx This class is in module, and it's only used by one component. Normally we can reference the model resource permission and pass via constructor. However, considering SubscriptionSender may be serialized, please try to use `Snapshot` instead.